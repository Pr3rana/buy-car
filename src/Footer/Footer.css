footer{
    position: absolute;
    font-size: 14px;
    width: 100%;
    height: 80px;
    display: flex;
    align-items: center;
    justify-content: center;
    border: 1px solid #ededed;
    bottom: 0;
    color: #4a4a4a;
}

