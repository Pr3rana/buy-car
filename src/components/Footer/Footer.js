import './Footer.css'
const Footer = () => {
    return ( 
        <>
            <footer className="cp-text">
                Â© Auto1 Group 2018
            </footer>
        </>
     );
}
 
export default Footer;
