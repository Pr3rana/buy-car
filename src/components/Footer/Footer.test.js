import { render, screen, cleanup } from '@testing-library/react';
import Footer from './Footer';

afterEach(cleanup);

describe("Footer render", ()=>{
    test('Initial Render', () => {
        const { container } = render(<Footer/>)
        expect(container.firstChild).toMatchSnapshot(`
        <footer data-testid ="footer" className="cp-text"></footer>
        `)
        expect(screen.queryByTestId('footer')).toBeInTheDocument()
    });
});

describe("Footer props",()=>{
    test("Re-render the Footer with different props",()=>{
        const { container } = render(<Footer footerBody="test footer body" />)
        expect(screen.getByTestId('footer')).toHaveTextContent('test');
        expect(container.firstChild).toMatchSnapshot(`
        <footer data-testid ="footer" className="cp-text">test footer body</footer>
        `)
    })
})