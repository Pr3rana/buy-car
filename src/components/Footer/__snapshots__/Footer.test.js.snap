// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer props Re-render the Footer with different props: 
        <footer data-testid ="footer" className="cp-text">test footer body</footer>
         1`] = `
<footer
  class="cp-text"
  data-testid="footer"
>
  test footer body
</footer>
`;

exports[`Footer render Initial Render: 
        <footer data-testid ="footer" className="cp-text"></footer>
         1`] = `
<footer
  class="cp-text"
  data-testid="footer"
/>
`;
